testutils/api/client.py

```diff
@@ -61,17 +64,19 @@ def call(
                 url = "http://localhost:8080/" + url.split("/", 3)[-1]
                 # wait a few seconds to let the port-forwarding fully initialize
                 time.sleep(3)
-            return requests.request(
-                method,
-                url,
-                json=body,
-                data=data,
-                params=qs_params,
-                headers=self.__make_headers(headers),
-                auth=auth,
-                verify=False,
-                files=files,
-            )
+            with warnings.catch_warnings():
+                warnings.simplefilter("ignore", category=InsecureRequestWarning)
```

Comment:
ok these two I do not understand, why are we having them here?

Reply:
The call to `requests.request` with `verify=False` generates a warning which we wish to ignore for the sake of keeping the logs nice and neat.